src/gatsby/rss/convertUrlsToAbsolute.mjs

```diff
@@ -0,0 +1,35 @@
+/**
+ * Converts relative URLs in markdown to absolute URLs.
+ *
+ * Handles:
+ * - Markdown links: [text](/path) -> [text](https://roadie.io/path)
+ * - Markdown images: ![alt](/path) -> ![alt](https://roadie.io/path)
+ * - HTML links: <a href="/path"> -> <a href="https://roadie.io/path">
+ * - HTML images: <img src="/path"> -> <img src="https://roadie.io/path">
+ *
+ * @param {string} markdown - The markdown content to process
+ * @param {string} [baseUrl='https://roadie.io'] - The base URL to use for absolute URLs
+ * @returns {string} - The markdown with relative URLs converted to absolute
+ */
+export function convertUrlsToAbsolute(markdown, baseUrl = 'https://roadie.io') {
+  if (!markdown || typeof markdown !== 'string') {
+    return markdown;
+  }
+
+  let result = markdown;
+
+  // Convert markdown links: [text](/path) -> [text](https://roadie.io/path)
+  // Matches [any text](url) where url starts with / but not //
+  result = result.replace(/\[([^\]]*)\]\(\/(?!\/)([^)]*)\)/g, `[$1](${baseUrl}/$2)`);
+
+  // Convert markdown images: ![alt](/path) -> ![alt](https://roadie.io/path)
+  result = result.replace(/!\[([^\]]*)\]\(\/(?!\/)([^)]*)\)/g, `![$1](${baseUrl}/$2)`);
+
+  // Convert HTML links: href="/path" -> href="https://roadie.io/path"
+  result = result.replace(/href=(["'])\/(?!\/)([^\1]*?)\1/g, `href=$1${baseUrl}/$2$1`);
+
+  // Convert HTML images: src="/path" -> src="https://roadie.io/path"
```

Comment:
Our blog posts and other content is written with the assumption that it will be rendered on the Roadie website. This means it has a lot of relative URLs in it (e.g. linking to `/backstage/plugins/`). These relative URLs won't work if the RSS feed is rendered in an RSS app. So I'm replacing them with absolute URLs.